library/snapshot.py

```diff
@@ -2033,438 +2239,217 @@ def get_json_from_args(args):
-    )
-
-    # Group parser
-    group_parser = argparse.ArgumentParser(add_help=False)
-    group_parser.add_argument(
-        "-g",
-        "--group",
-        nargs="?",
-        action="store",
-        required=False,
-        default=None,
-        dest="set_json",
-    )
-
-    # LVM VG/LV parser
-    lvm_parser = argparse.ArgumentParser(add_help=False)
-    lvm_parser.add_argument(
-        "-vg",
-        "--volumegroup",
-        nargs="?",
-        action="store",
-        default=None,
-        dest="volume_group",
-        help="volume group to snapshot",
-    )
-    lvm_parser.add_argument(
-        "-lv",
-        "--logicalvolume",
-        nargs="?",
-        action="store",
-        default=None,
-        dest="logical_volume",
-        help="logical volume to snapshot",
-    )
-
-    # arguments for operations that do verify
-    verify_parser = argparse.ArgumentParser(add_help=False)
-    verify_parser.add_argument(
-        "-v",
-        "--verify",
-        action="store_true",
-        default=False,
-        dest="verify",
-        help="verify VGs and LVs have snapshots",
-    )
-
-    # arguments for operations that deal with required space
-    req_space_parser = argparse.ArgumentParser(add_help=False)
-    # TODO: range check required space - setting choices to a range makes the help ugly.
-    req_space_parser.add_argument(
-        "-r",
-        "--requiredspace",
-        dest="required_space",
-        required=False,
-        type=int,  # choices=range(10,100)
-        default=20,
-        help="percent of required space in the volume group to be reserved for snapshot",
-    )
-
-    # arguments for operations that deal with mount of filesytems
-    mountpoint_parser = argparse.ArgumentParser(add_help=False)
-    mountpoint_parser.add_argument(
-        "-m",
-        "--mountpoint",
-        dest="mountpoint",
-        required=False,
-        type=str,
-        help="mount point for block device",
-    )
-
-    # arguments for operations that deal with mount of filesytems
-    mount_parser = argparse.ArgumentParser(add_help=False)
-    mount_parser.add_argument(
-        "-b",
-        "--blockdev",
-        dest="blockdev",
-        required=False,
-        type=str,
-        help="mount point for block device",
-    )
-    mount_parser.add_argument(
-        "-t",
-        "--type",
-        dest="fstype",
-        required=False,
-        default="",
-        type=str,
-        help="filesystem type",
-    )
-    mount_parser.add_argument(
-        "-o",
-        "--options",
-        dest="options",
-        required=False,
-        type=str,
-        help="mount options",
+def run_module():
+    logger.info("run_module()")
+    vg_include = None
+    # define available arguments/parameters a user can pass to the module
+    # available arguments/parameters that a user can pass
+    module_args = dict(
+        snapshot_lvm_action=dict(type="str"),
+        ansible_check_mode=dict(type=bool),
```

Comment:
```suggestion
        ansible_check_mode=dict(type="bool"),
```

Reply:
Actually, I don't think you need to define a parameter for this since you specify `supports_check_mode=True` in the AnsibleModule constructor